docs/site/DEVELOPING.md

```diff
@@ -113,6 +114,30 @@ npm script `lint:fix`.
 ```sh
 npm run lint:fix
 ```
+## File naming convention
+
+For consistency, we follow
+[Angular's file naming convention](https://angular.io/guide/styleguide#separate-file-names-with-dots-and-dashes).
+It also helps to derive the usage of files by inspecting the names.
+Besides our LoopBack 4 codebase, we also follow this naming convention in our generated artifacts from the CLI tooling:
+`{name}`.`{artifact-type}`.ts
+
+Examples are:
+```
+src/decorators/authenticate.decorator.ts
+src/boot.component.ts
+```
+
+In addition, the test files under `test` folder are categorized
```

Comment:
I think this might read better:
`In addition, files in the `test` directory are categorized` ...

Reply:
`folder` or `directory`? I think `folder` is more commonly used in our documentation?